ugvc/mrd/featuremap_utils.py

```diff
@@ -211,12 +217,24 @@ def __init__(
         self.CYCLE_SKIP_FLAG = FeatureMapFields.IS_CYCLE_SKIP.value
         self.PREV_N_BP = f"prev_{self.motif_length_to_annotate}bp"
         self.NEXT_N_BP = f"next_{self.motif_length_to_annotate}bp"
+        self.NEXT_1 = "next_1sbp"
```

Comment:
This can also be reduced to a loop I think

Reply:
Agree, please use the input arg motif_length_to_annotate instead of hard coding 3

Reply:
done